libcxx/test/std/experimental/reflection/reflect-invoke.verify.cpp

```diff
@@ -0,0 +1,65 @@
+//===----------------------------------------------------------------------===//
```

Comment:
Great idea to add these!!

We _desperately_ need more of these "fail-tests" for reflection (I haven't written any) - after you're finishing with this PR, if you have any interest in adding some, that'd be a huge help.

Reply:
I don't mind to work on it!

It would be helpful to me if you could elaborate what  testing could be useful for authors of paper and users of this fork.

Should it be only different tests to trigger diagnostic messages or I need to try to cover some branches where we just say that expression is invalid (`return true;`)?

Reply:
Both would be great - we don't need full coverage of "expression is invalid" cases for every function, but a few would be nice.

Reply:
I see, will do after this PR 👍🏻